doc/source/rllib/package_ref/replay_buffers.rst

```diff
@@ -0,0 +1,164 @@
+.. _replay-buffer-api-reference-docs:
+
+Replay Buffers
+##############
+
+Quick intro into replay buffers in RL
+=====================================
+
+When we talk about replay buffers in reinforcement learning, we generally mean a buffer that stores and replays experiences.
+In python, a simple buffer can be implemented by a list to which elements are added and sampled from.
+Such buffers are used mostly in offline learning algorithms. This makes sense intuitively because these algorithms can learn from
+experiences that are stored in the buffer but where produced by a previous version of the policy (or even a completely different policy).
+
+Sampling Strategy
+-----------------
+
+When sampling from a replay buffer, we choose which experiences to train our agent with. A straightforward strategy to choose these
+samples that has proven effective for many algorithms is uniform sampling. That is, choosing a random item from the buffer each time we sample.
+A strategy that has proven better in many cases is `Prioritized Experiences Replay (PER) <https://arxiv.org/abs/1511.05952>`__.
+In PER, experiences are assigned a priority which denotes their significance, or in simple term, how much we expect
```

Comment:
"In PER, single items in the buffer are assigned ..."
...
"to learn from these items".